### ISSUE TITLE
[Blackhole map] Blackhole from Urduliz not working

### ISSUE THREAD

**Issue by iblanco-s** [demostanis/42evaluators#4]

**Comment by demostanis** [comment]
yeah, im pretty sure that's because of a recent change i made with trailing slashes in endpoint, which changed http requests with relative paths...
i fixed that and deployed it
can you check it works? (you might have to clear the cache)

**Comment by demostanis** [comment]
too much of the window? what's your window width?

**Comment by iblanco-s** [comment]
on imac 27
